magento/ece-tools
# MAGECLOUD-3611: Multi-threaded SCD Doesn’t Work as Expected

### Description
* Customers now observe issues with SCD, which are common across multiple projects

### Fixed Issues (if relevant)

1. magento/ece-tools#503: [ECE-Tools] Multi-threaded SCD Doesn’t Work as Expected
2. MAGECLOUD-3611

### Manual testing scenarios

1. prepare configurations to make SCD runs with 4 and more threads
2. SCD_THREADS: 4
    SCD_MATRIX:
        "magento/backend":
            language:
                - en_US
                - fr_FR
                - ja_JP
                - sk_SK
                - nb_NO
                - en_GB
                - uk_UA
                - es_ES
                - lv_LV
                - it_IT
        "magento/luma":
            language:
                - en_US
                - fr_FR
                - ja_JP
                - sk_SK
                - nb_NO
                - en_GB
                - uk_UA
                - es_ES
                - lv_LV
3. SCD should be run as on build as on deploy phase

### Contribution checklist
